modules/wordpress-com-compose.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Module Name: WordPress.com Block Editor
+ * Module Description: Allow new block editor posts to be composed on WordPress.com.
+ * Jumpstart Description: Allow new block editor posts to be composed on WordPress.com.
+ * Sort Order: 15
+ * First Introduced: 7.2
+ * Requires Connection: Yes
+ * Auto Activate: No
+ * Module Tags: Writing
+ * Feature: Writing
+ * Additional Search Queries: iframes, allow, compose, WordPress.com, block, editor, post
+ */
+
+function jetpack_disable_send_frame_options_header() {
+	if ( jetpack_framing_allowed() ) {
+		remove_action( 'admin_init', 'send_frame_options_header' );
+	}
+}
+add_action( 'admin_init', 'jetpack_disable_send_frame_options_header', 1 ); // High priority to get ahead of send_frame_options_header
+
+function jetpack_get_frame_nonce() {
```

Comment:
This function does not seem to be used either

Reply:
Good point – removed in 159a684bd.